src/provider/constants.ts

```diff
@@ -23,35 +23,53 @@ export const Timeouts = {
     WINDOW_SEEN_TO_REGISTERED: 5000
 };
 
-export const DESKTOP_CHANNELS = [
+export const SYSTEM_CHANNELS = [
     {
         id: 'red',
-        name: 'Red',
-        color: 0xFF0000
+        visualIdentity: {
+            name: 'Red',
+            color: '#FF0000',
+            glyph: 'https://openfin.co/favicon.ico'
+        }
```

Comment:
Thinking forward to when we make this desktop-configurable (since this is essentially temporary, until we replace this with a config store query), would we want `visualIdentity` within the config schema, or the previous "flat" style?